src/common/msg.cc

```diff
@@ -334,7 +334,11 @@ extern "C" {
 		msg_expand(text, line, filename, funcname, simplefunc, LOG_CRIT, fmt, &args);
 		va_end(args);
 
+#ifdef FUZZ_TARGET
```

Comment:
Agreed, this could be useful for testing.
Instead of calling this the specific to testing FUZZ_TARGET, perhaps EXIT_ON_EXCEPTION which describes what it does better?

Reply:
I like the idea. Changed accordingly